db/migrations/20200430103648_add_non_canonical_column_to_storage_diff.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+ALTER TABLE public.storage_diff
```

Comment:
Totally up for this, but I'm not 100% sure how to structure it. I think we probably would want to use a partial index, since it's a boolean - but it also seems like a partial index on `is false`, while relevant for the query, would capture a big majority of the rows on the table 🤔 

I am hoping that the `GetNewDiffs` query will be relatively performant even without an index on the column, since we have access to the partial index on `checked is false`. But definitely on board for ways to optimize further since we are adding a condition to the query

Reply:
Oh that's true that the partial index would be on `is false` which would be a lot of the diffs. I'm cool with going forward with how it is now, and just keeping in mind that a partial may be helpful in the future.